PURL-TYPES.rst

```diff
@@ -87,6 +87,25 @@ bitnami
       pkg:bitnami/wordpress@6.2.0
       pkg:bitnami/wordpress@6.2.0?arch=arm64
 
+brew
+----
+``brew`` for Homebrew-based packages:
+
+- The default repository (tap) is ``homebrew/core``.
+
+  - The tap syntax is ``https://github.com/{org}/Homebrew-{tap}``, so
+    ``homebrew/core`` corresponds to the URL ``https://github.com/homebrew/homebrew-core``.
```

Comment:
**suggestion (non-blocking):** 
```suggestion
  - The typical tap identifier expands to the URL ``https://github.com/{org}/Homebrew-{tap}``, so the tap identifier
    ``homebrew/core`` corresponds to the tap URL ``https://github.com/homebrew/homebrew-core``.
```

I'm maybe too wordy here, but for a spec, wordy might be good.

Reply:
Agreed, wordy is good!